roles/edpm_bootstrap/tasks/download_cache.yml

```diff
@@ -34,3 +34,15 @@
   until: edpm_bootstrap_release_version_package_download.rc == 0
   retries: "{{ edpm_bootstrap_download_retries }}"
   delay: "{{ edpm_bootstrap_download_delay }}"
+
+- name: Download updated packages
+  become: true
+  ansible.builtin.dnf:
+    name: "*"
```

Comment:
have you measured the execution time? the last time I've  checked it (with much older ansible versions, I must admit) that command was taking hours compared to minutes by dnf update

Reply:
yea, that was quite ago... https://github.com/ansible/ansible/issues/26995
the things might get improved since then

Reply:
if still takes "hours" we should raise a huge issue on ansible core team :)

Reply:
ok, please change for the shell command then, until fixed

Reply:
Sure, when someone will demonstrate with proper testing that `ansible.builtin.dnf` is much slower than `dnf` called in a `command`/`shell` task, we'll change every instance of it in the repo and raise a bug to the ansible team.

Reply:
I checked on my env at hand, the time it takes looks pretty much the same in both cases:

dnf module:
```
real    1m9.585s                            
user    0m43.438s
sys     0m11.120s
```
shell command:
```
real    1m4.617s
user    0m45.630s
sys     0m10.170s
```

Reply:
we'll keep this aspect monitored though (especially with many repository enabled and lot of packages to install/upgrade)